orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -17,22 +20,30 @@ class OWDifference(widget.OWWidget):
                   '1st or 2nd order discrete difference along its values. '
     icon = 'icons/Difference.svg'
     priority = 570
+    keywords = ['difference', 'derivative', 'quotient', 'percent change']
 
     class Inputs:
         time_series = Input("Time series", Table)
 
     class Outputs:
         time_series = Output("Time series", Timeseries)
 
+    class Operation(IntEnum):
+        DIFFERENCE = 0
+        QUOTIENT = 1
+        PERCENT = 2
+
     want_main_area = False
     resizing_enabled = False
+    op_names = dict(enumerate(['diff', 'quot', 'perc']))
```

Comment:
If you make **`Operation(StrEnum)`**, you can then just use that instead. Also makes sense since they are unorderable, as well as non-negatable, operations.

Reply:
True, but that means i can't compare it with ints, which is what combo box returns. Unless I'm missing something, I have to make an awkward mapping somewhere. I changed it to perhaps less awkward though.

Reply:
I think that's maybe what [`sendSelectedValue=True`](https://github.com/biolab/orange-widget-base/blob/0ea72bc134dded42abe82f1ff025bcd61f507639/orangewidget/gui.py#L1434-L1437) was meant for.

Reply:
Thanks, I missed that one.